Makefile

```
@@ -1,8 +1,24 @@
+# copy .env-example to .env and modify contents
+include .env
+
+BUILD_DIR := build/tmp/deploy/images/iot2000/
+BUILD_FILE := core-image-minimal-iot2000.wic
+BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
+HASH := $(shell git rev-parse HEAD)
+DATE := $(shell date +%Y%m%d-%H%M%S)
```

Shorter date format. We can still keep track image versions by branch and short SHA
```suggestion
DATE := $(shell date +"%Y-%m-%d")
```